test/testDateTimeGroup.jsx

```diff
@@ -0,0 +1,271 @@
+'use strict';
+
+var React = require('react/addons');
+var TestUtils = React.addons.TestUtils;
+
+var TimePicker = require('react-time-select');
+var DatePicker = require('react-datepicker');
+var moment = require('moment');
+
+var expect = require('chai').expect;
+var sinon = require('sinon');
+
+var shallowRender = require('react-shallow-render');
+
+describe('DateTimeGroup', function() {
+  var clock, DateTimeGroup;
+
+  before(function() {
```

Comment:
`before` only happens once, which maps more closely to the behaviour of `require`. If this used `beforeEach` we'd have to clear the module cache each time for things to work as expected.